docs/install/windows_setup.md

```diff
@@ -212,7 +213,22 @@ These steps are required after building from source. If you already installed MX
 
 1. Install ```Python``` using windows installer available [here](https://www.python.org/downloads/release/python-2712/).
 2. Install ```Numpy``` using windows installer available [here](https://scipy.org/index.html).
-3. Next, we install Python package interface for MXNet. You can find the Python interface package for [MXNet on GitHub](https://github.com/dmlc/mxnet/tree/master/python/mxnet).
+3. Start ```cmd``` and create a folder named ```common```(```mkdir C:\common```)
+4. Download the [mingw64_dll.zip](https://sourceforge.net/projects/openblas/files/v0.2.12/mingw64_dll.zip/download), unzip and copy three libraries (.dll files) that openblas.dll depends on to ```C:\common```.
+5. Copy the required .dll file to ```C:\common``` and make sure following libraries (.dll files) in the folder.
+```
+libgcc_s_seh-1.dll (in mingw64_dll)
+libgfortran-3.dll (in mingw64_dll)
+libquadmath-0.dll (in mingw64_dll)
+libopenblas.dll (in OpenBlas folder you download)
+opencv_world341.dll (in OpenCV folder you download)
+```
+6. Add ```C:\common``` to Environment Variables.
+ * Type ```control sysdm.cpl``` on ```cmp```
+ * Select the **Advanced tab** and click **Environment Variables**
+ * Double click the **Path** and click **New**
+ * Add ```C:\common``` and click OK
+7. Next, we install Python package interface for MXNet. You can find the Python interface package for [MXNet on GitHub](https://github.com/dmlc/mxnet/tree/master/python/mxnet).
```

Comment:
There's a section on the page that tells the user what to do next for the Python bindings. Lead them to that!
https://mxnet.incubator.apache.org/install/windows_setup.html#install-mxnet-for-python
Also, I see that the anchor was changed so that the Table of Contents link to the Python section is broken (likely my fault). Please update that since you're already updating this file. https://github.com/apache/incubator-mxnet/blame/master/docs/install/windows_setup.md#L13